### DISCUSSION TITLE
Informações das escolas do Brasil

### DISCUSSION THREAD

**Discussion by clewertonx1** [BrasilAPI/BrasilAPI#217] [Ideas]
Opa galera, 
Eu nunca mexi com nextJS e também não tenho muito conhecimento em backend, mas, apesar da falta de conhecimentoKK estou tentando adicionar a funcionalidade de busca a escolas de todo o Brasil, eu estou usando a api de dados abertos do INEP ( http://educacao.dadosabertosbr.com/api ), bom eu to literalmente com a mão na massa, criei um module e upei no npm que já tem a função de buscar as escolas por nome, porem, não sei como integrar. Enfim, se alguém tiver afim de me ensinar eu iria agradecerKK

Para instalar o module 
npm install escolas-brasil
No github: 
https://github.com/clewertonx1/escolas-brasil

meu telegram:
t.me/clewerton

Ah e eu nunca contribuir com nada no github, entao eu to aprendendo, nem sei se pode fazer esse tipo de issueKKKK

**Comment by lucas123ho** [comment]
Massa @clewertonx1 ! Se quiser posso tentar te ajudar.

**Comment by clewertonx1** [comment]
Opa cara, se tiver com alguma ideia pra impletar na api ja vai construindo. Vou deixar primeiro a API bem completa pra depois tentar implementar no BrasilAPI. 
Me chama no telegram que a gente vai conversando blz ?
